### DISCUSSION TITLE
Supabase studio and functions

### DISCUSSION THREAD

**Discussion by magick93** [supabase/supabase#7946] [Questions] [answered]
Hello

When I run the supabase studio locally, using either docker-compose or the cli there is no functions or triggers option in studio view.

How can I enable this functionality?

**Comment by pixtron** [comment, marked as answer]
I think you can't currently enable this functionality. 

> The Functions interface is coming soon.

At least thats one of the points listed under "Limitations" in [Local Development](https://supabase.com/docs/guides/local-development#limitations)

**Reply by magick93** [reply]
Yeah, I'm assuming there is a setting somewhere to enable them, as they are available in the hosted version.
